src/pages/graphql/schema/store/queries/recaptcha-form-config.md

```diff
@@ -0,0 +1,94 @@
+---
+title: recaptchaFormConfig query
+edition: pwa
+---
+
+# recaptchaFormConfig query
+
+The `recaptchaFormConfig` query returns information about reCaptcha configuration for selected form type.
+
+## Syntax
+
+`recaptchaFormConfig(formType: ReCaptchaFormEnum!): ReCaptchaConfigOutput`
+
```

Comment:
Add the following lines below:

## Reference

The `recaptchaFormConfig` reference provides detailed information about the types and fields defined in this mutation.